web/client/plugins/__tests__/pluginsTestUtils.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2019, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import React from 'react';
+import endsWith from 'lodash/endsWith';
+import {Provider} from 'react-redux';
+
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import { combineEpics, createEpicMiddleware } from 'redux-observable';
+
+const createRegisterActionsMiddleware = (actions) => {
+    return () => next => action => {
+        actions.push(action);
+        return next(action);
+    };
+};
+
+export const getPluginForTest = (pluginDef, storeState, plugins) => {
```

Comment:
Looks a very good utility. Can you jsdoc them?

Reply:
it's going to be documented in the dev plugins documentation